nx/source/services/fs.c

```diff
@@ -633,6 +700,29 @@ Result fsIsExFatSupported(bool* out)
     return rc;
 }
 
+// Wrapper(s) for fsCreateSaveDataFileSystemBySystemSaveDataId.
+Result fsCreate_SystemSaveDataWithOwner(FsSaveDataSpaceId space_id, u64 save_data_id, u64 user_id, u64 owner_id, u64 size, u64 journal_size, u32 flags) {
+    FsSave save;
+    FsSaveCreate create;
+
+    memset(&save, 0, sizeof(save));
+    save.userID = (u128)user_id | ((u128)owner_id << 64);
```

Comment:
Does this really match official sw?(Is this the account userID?)

Reply:
I mistakenly had the size of userid as a u64 in my local db so the two looked like they were getting clobbered together, but I've fixed both now.